kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -282,10 +283,16 @@ class Transport(val options: TransportOptions) {
 //            }
 //        }
 
-        val result = runBlocking {
-            SDKResponse.SDKSuccessResponse(client.call(builder).response.receive<T>())
+        var result: SDKResponse
```

Comment:
You can still use a `val` here I believe. You should be able to do:
```
val result = try {
            runBlocking {
                SDKResponse.SDKSuccessResponse(client.call(builder).response.receive<T>())
            }
        } catch (e: Exception) {
            SDKResponse.SDKErrorResponse("$method $path $e")
        } finally {
            client.close()
        }
```

https://kotlinlang.org/docs/reference/exceptions.html#try-is-an-expression